### PR TITLE
styling: wishlist & logout page

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes are to be made

Added - Wishlist & Logout page
wishlsit

GIFY -

### CODE REVIEW

File: Pages/Checkout/checkout.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.doc-success{
+    display: flex;
+    align-items: center;
+    justify-content: center;
```
Comment (haarsh24) [inline Pages/Checkout/checkout.css:4]: Suggestion:
You can declare these properties in util and use it as class, as it can be used in multiple areas.

**Review by haarsh24 — APPROVED** [review]
Code looks good , you can merge.

**Review by abhishek17551 — APPROVED** [review]
suggest to add some styling for the buttons either when hovered or clicked. rest all  looks fine.

File: Pages/Checkout/checkout.css (hunk 1 above)
Comment (irajbhinde) [reply Pages/Checkout/checkout.css:4]: Noted, thanks!

**Pull request merged**